src/pages/index.md

```diff
@@ -3,11 +3,13 @@ title: Commerce PHP Extensions
 description: Review resources for developing PHP-based extensions for Adobe Commerce and Magento Open Source.
 ---
 
-<Hero slots="heading, text" background="rgb(244, 111, 37)"/>
+<Hero slots="image, heading, text"/>
+
+![Commerce PHP Extensions](_images/home-bg.jpeg)
 
 # Commerce PHP Extensions
 
-Lorem ipsum dolor sit amet
+The resources listed here contain information for developers who want to cusomtize or extend the Adobe Commerce and Magento Open Source applications. You can also create extensions that introduce new functionality and distribute them to merchants.
```

Comment:
```suggestion
The resources listed here contain information for developers who want to customize or extend the Adobe Commerce and Magento Open Source applications. You can also create extensions that introduce new functionality and distribute them to merchants.
```
I think there's too much text for the Hero block. I would simplify it to something like

Learn how to create and maintain PHP extensions for Adobe Commerce and Magento Open Source.

If that's too terse, then at least fix the typo. 😃 

I can't comment on the Resources section directly since it wasn't changed in this PR, but you can drop the Contributor statistics link. This is an early version of the page, so it can wait till later.